### PR TITLE
Add more development tools

### PR DESCRIPTION
**Pull request by lumag**
Add several tools used on a day-to-day basis during development (fastrpc, ALSA and kmscube)

### CODE REVIEW

File: recipes-test/images/initramfs-test-image.bb [hunk 1]
```diff
@@ -3,6 +3,10 @@ require recipes-test/images/initramfs-tiny-image.bb
 DESCRIPTION = "Small ramdisk image for running tests (bootrr, etc)"
 
 PACKAGE_INSTALL += " \
+    alsa-utils-amixer \
+    alsa-utils-aplay \
+    alsa-utils-alsaucm \
```
Comment (ndechesne) [inline recipes-test/images/initramfs-test-image.bb:8]: alphabetical order?

File: recipes-test/images/initramfs-test-image.bb [hunk 2]
```diff
@@ -31,6 +31,7 @@ PACKAGE_INSTALL += " \
     tqftpserv \
     usbutils \
     wpa-supplicant \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'kmscube mesa-megadriver', '', d)} \
```
Comment (ndechesne) [inline recipes-test/images/initramfs-test-image.bb:34]: mesa-megadriver doesn't really belong here, no? it's more a MACHINE conf. why is it here?

Comment (lumag) [reply recipes-test/images/initramfs-test-image.bb]: Removed

File: recipes-test/images/initramfs-test-image.bb (hunk 1 above)
Comment (lumag) [reply recipes-test/images/initramfs-test-image.bb]: fixed

**Pull request merged**